spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Ingredients index page", type: :feature do
+  before :each do 
+    @ingredient1 = Ingredient.create!(name: "Onions", cost: 4) 
+    @ingredient2 = Ingredient.create!(name: "Yeast", cost: 2) 
+    @ingredient3 = Ingredient.create!(name: "Beans", cost: 3) 
+  end
+
+  describe "As a visitor to /ingredients" do 
+    it "Shows a list of all ingredients and price" do 
+      visit '/ingredients'
+
+      expect(page).to have_content("Onions")
```

Comment:
This test is good, but it could be more robust with the use of within blocks to verify that, for example, the onion price is within the section for onions, and is not just a floating `4` integer somewhere else on the page.